Dispatcher/UI/Screens/HomePage/HomePageViewController.swift

```diff
@@ -0,0 +1,58 @@
+import UIKit
+
+class HomePageViewController: UIViewController {
+    
+    @IBOutlet weak var tableView: UITableView!
+    var viewModel: HomePageViewModel = HomePageViewModel(repository: MockArticleRepository())
+    var mockArticles:[Card] = []
+    let navigationBar = NavigationBar()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        mockArticles = viewModel.repository.getArticles()
+        initNavigationBar()
+        initTableView()
+    }
+    
+    func initNavigationBar() {
+        navigationBar.delegate = self
+        navigationBar.setupNavigationBar(for: self)
+    }
+    
+    func initTableView() {
+        tableView.dataSource = self
+        tableView.delegate = self
+        tableView.register(UINib(nibName:String(describing: ArticleCell.self), bundle: nil), forCellReuseIdentifier: ArticleCell.identifier)
+        tableView.rowHeight = 449
```

Comment:
we do not do that
there is a special function for it called heightForRowAt in UITableViewDelegate